src/lib.rs

```diff
@@ -19,12 +19,14 @@ pub fn cli_print_error_chain<'a>(
         &'a (dyn Error + 'static),
     ) -> Option<String>,
 ) {
-    let msg = format!("Error: {component} exited with the following");
+    error!("Debug Info: {top_error:?}");
```

Comment:
nit: "Debug Info" now feels inappropriate. How about "Fatal Error:" - the fact this is the debug output should be obvious.